src/main/kotlin/org/rust/openapiext/utils.kt

```diff
@@ -419,16 +419,16 @@ fun <T, D> getCachedOrCompute(
     return value
 }
 
-inline fun <R> nonBlocking(project: Project, crossinline block: () -> R, crossinline uiContinuation: (R) -> Unit) {
+inline fun <R> Project.nonBlocking(crossinline block: () -> R, crossinline uiContinuation: (R) -> Unit) {
```

Comment:
Looks like this method is intended to be used only from the UI thread. But I suppose it might be unclear when you don't look into its implementation. Maybe add a doc comment like "Must be invoked on EDT"?

Reply:
Added a comment